client/src/common/client.ts

```diff
@@ -348,9 +348,9 @@ InlineCompletionMiddleware & TextDocumentContentMiddleware & GeneralMiddleware;
 export type LanguageClientOptions = {
 	documentSelector?: DocumentSelector | string[];
 	diagnosticCollectionName?: string;
-	outputChannel?: OutputChannel;
+	outputChannel?: LogOutputChannel | OutputChannel;
```

Comment:
Writing `LogOutputChannel | OutputChannel` instead of `OutputChannel` isn't really necessary but I did it so that the log output channel is more prominently mentioned in the source code. I can undo this if undesired.